Jenkinsfile

```diff
@@ -5,13 +5,15 @@ pipeline {
         stage('Git checkout') {
             steps {
                 git 'https://github.com/linagora/openpaas-james'
+                sh 'git submodule init'
+                sh 'git submodule update'
             }
         }
+        stage('Compile') {
+            sh 'mvn install -Dmaven.javadoc.skip=true -DskipTests -T1C'
+        }
         stage('Test') {
             steps {
-                sh 'git submodule init'
-                sh 'git submodule update'
-                sh 'mvn install -Dmaven.javadoc.skip=true -DskipTests -T1C'
                 dir("openpaas-james") {
                     sh 'mvn -B test'
```

Comment:
test -> surefire:test ?

Reply:
what is the subtility here?

Reply:
test runs validate stage, and do some extra work while surefire:test just plays tests

Reply:
Nice thanks for the tip